portfolio/src/test/java/com/google/sps/ShortestTaskFirstTest.java

```diff
@@ -0,0 +1,187 @@
+package com.google.sps.data;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collection;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class ShortestTaskFirstTest {
+  private static Instant BEGINNING_OF_DAY = Instant.parse("2020-06-25T00:00:00Z");
+  private static Instant END_OF_DAY = Instant.parse("2020-06-25T23:59:59Z");
+  private static Instant THREE_DAYS_LATER = Instant.parse("2020-06-28T00:00:00Z");
+
+  private static Duration TIME_15_MINUTES = Duration.ofSeconds(900);
```

Comment:
Consider replacing 900 with 15 * 60 for readability. Here and below.

Reply:
Also, I think the Duration variables can be named DURATION_* instead of TIME_*. That way we can easily distinguish duration variables from instant variables.

Reply:
Done.